ApollosProject/apollos-prototype
# Move UserAvatar to connected package

## DESCRIPTION
![image](https://user-images.githubusercontent.com/2528817/73286898-fc8dd780-41bd-11ea-97dd-1339fd8ce1bb.png)

### What does this PR do, or why is it needed?
Moves relevant UserAvatar components to connected package.

### What design trade-offs/decisions were made?
I left the UserAvatarHeader component in the church app with plans to refactor it more in a follow-up PR.

### How do I test this PR?
* Go to the connect tab and verify the profile image is working
* Click on the setting icon
* User's should see their profile image and a camera icon.
* Click the camera icon and verify updating the user profile image flow works are expected.

### What automated tests did you write?
I wrote a snapshot test for the UserAvatarConnected that should mock the fetching of the user profile image.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
